pwscf/v6.3/pw2wannier90.f90

```diff
@@ -3475,9 +3480,9 @@ SUBROUTINE compute_amn_with_scdm
       DO jpt = 0,dffts%nr2-1 
          DO ipt = 0,dffts%nr1-1
             lpt = lpt + 1
-            rpos(lpt,1) = REAL(ipt)/dffts%nr1 
-            rpos(lpt,2) = REAL(jpt)/dffts%nr2 
-            rpos(lpt,3) = REAL(kpt)/dffts%nr3 
+            rpos(lpt,1) = DBLE(ipt)/DBLE(dffts%nr1)
```

Comment:
I think it would be better to use REAL(ipt,DP) here, rather than use DBLE. To my mind DBLE is a more fortran77 way of handling precision.

Reply:
Thank you for the comments. I have changed to them to REAL(*, DP). (Both for v6.3 and v6.4)